.travis.yml

```diff
@@ -26,23 +26,23 @@ cache:
     - $HOME/.composer/cache/files
 
 env:
-  matrix: SYMFONY_VERSION=4.0.*
+  matrix: SYMFONY_VERSION=4.2.*
   global:
-    - SYMFONY_DEPRECATIONS_HELPER="/.*each.*/"
+    - SYMFONY_DEPRECATIONS_HELPER="weak"
     - SYMFONY_PHPUNIT_DIR=.phpunit SYMFONY_PHPUNIT_REMOVE="symfony/yaml"
-    - SYMFONY_PHPUNIT_VERSION=6
-    - TEST_INSTALLATION=false
```

Comment:
is this now the default? we should only test the installation in one build, not all of them

Reply:
i think the intention was that we would switch it to fail the build once it was reliable enough. in the beginning we did it for supporting symfony 4 and it failed everywhere because we did not support it yet.

Reply:
i put it back.